handlers-api/build.gradle

```diff
@@ -1,34 +1,9 @@
-apply plugin: 'info.solidsoft.pitest'
-
-jar {
-  enabled = true
-  archiveBaseName = 'civil-sdt-handlers-api'
-}
-
-dependencyCheck {
-    suppressionFile = "${rootDir}/config/owasp/suppressions.xml"
-}
-
 dependencies {
     implementation project(":domain")
     implementation project(":utils")
     implementation project(":producers-api")
-    implementation project(":validators")
-    testImplementation project(":domain").sourceSets.unitTest.output
-    testImplementation project(":utils").sourceSets.unitTest.output
-    testImplementation project(":producers-api").sourceSets.unitTest.output
-    testImplementation project(":validators").sourceSets.unitTest.output
-
-    testImplementation project(":domain").sourceSets.integTest.output
-    testImplementation project(":utils").sourceSets.integTest.output
-    testImplementation project(":producers-api").sourceSets.integTest.output
-    testImplementation project(":validators").sourceSets.integTest.output
-}
-
-bootJar {
-    enabled = false
-}
 
-sonarqube {
-  skipProject = true
-}
+    testImplementation project(path: ':domain', configuration: 'testOutput')
+    testImplementation project(path: ':producers-api', configuration: 'testOutput')
+    testImplementation project(path: ':utils', configuration: 'testOutput')
```

Comment:
The handlers-api module doesn't have any tests so these testImplementation lines may not be needed.

Reply:
good point, removed those

Reply:
Confirmed